### PR TITLE
Support provisioning Wirnet iStation

### PR DESCRIPTION
**Pull request by rvolosatovs**
#### Summary

Adds support for provisioning Wirnet iStation.

#### Changes

- Support Wirnet iStation

#### Notes for Reviewers

Massive thank you for assistance and trust from Skylab!

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Testing: The changes are tested manually.
- [x] Documentation: Relevant documentation is added or updated.

### CODE REVIEW

**Comment by johanstokking** [comment]
What is the status? Is this already being tested with real gateways by the community?

**Comment by rvolosatovs** [comment]
I tested this myself on a Wirnet iStation provided by Skylab.

File: provision.sh [hunk 1]
```diff
@@ -256,92 +263,142 @@ function provision {
     local stackAddr="${2}"
     local gatewayID="${3}"
     local apiKey="${4}"
+    local gatewayModel="${5:-"${wirnetStationModel}"}"
```
Comment (rvolosatovs) [inline provision.sh]: IMO this may be confusing for some, @johanstokking can we break the API here and require a 5th argument(model)?

**Comment by rvolosatovs** [comment]
After this is merged I will tag `0.0.1` and open a PR to lorawan-stack using versioned script.

**Review by johanstokking — APPROVED** [review]
Great stuff

It would be nice if you can add a brief `CONTRIBUTING.md` with some instructions on what to do if Kerlink updates their stuff, i.e. links and hashes to put in our script

**Comment by rvolosatovs** [comment]
Filed https://github.com/TheThingsNetwork/kerlink-wirnet-firmware/issues/12

**Pull request merged**